File: horners_method/javascript/horners_method.js
const hornersMethod = (polynomArr, n, k) => {
    result = polynomArr[0];
    for(let i = 1; i < n; i++) {
        result = result * k + polynomArr[i]
    }

    console.log(`${k} => ${result}`)
}

arr = [2, -6, 3, -4]

hornersMethod(arr, arr.length, 5)

File: horners_method/python/horners_method.py
def hornersMethod(polynomArr, n, k):
    result = polynomArr[0]
    for i in range(1, n):
        result = result * k + polynomArr[i]
    
    print(f"{k} => {result}")

arr = [2, -6, 3, -4]

hornersMethod(arr, len(arr), 5)